--- origsrc/iperf-2.0.4/CYGWIN-PATCHES/README	1969-12-31 19:00:00.000000000 -0500
+++ src/iperf-2.0.4/CYGWIN-PATCHES/README	2010-03-16 08:27:17.710809000 -0400
@@ -0,0 +1,58 @@
+iperf
+------------------------------------------
+Internet Protocol bandwidth measuring tool
+
+Runtime requirements:
+  cygwin-1.7.1-1 or newer
+  libgcc1
+  libstdc++6
+  
+Build requirements:
+  cygwin-1.7.1-1 or newer
+  gcc-3.4.4-1 or newer
+  g++-3.4.4-1 or newer
+
+Canonical homepage:
+  http://iperf.sourceforge.net
+
+Canonical download:
+  http://sourceforge.net/projects/iperf/files
+
+License:
+  Univerity of Illinois BSD-like.  See doc/ui_license.
+
+Language:
+  C, C++
+
+------------------------------------
+
+Build instructions:
+  unpack iperf-2.0.4-1-src.tar.bz2
+  if you use setup to install this src package, it will be
+	 unpacked under /usr/src automatically
+  cd /usr/src
+  cygport iperf-2.0.4-1.cygport all
+
+This will create:
+  /usr/src/iperf-2.0.4-1.tar.bz2
+  /usr/src/iperf-2.0.4-1-src.tar.bz2
+
+Or use 'cygport iperf-2.0.4-1.cygport prep' to get a patched source directory
+
+To find out the files included in the binary distribution, you can
+use "cygcheck -l iperf-2.0.4-1", or browse the listing for the appropriate version
+at <http://cygwin.com/packages/>.
+
+------------------
+
+Port Notes:
+Straight compile.
+
+----------  iperf-2.0.4 -- 16 March, 2010 -----------
+Initial release
+
+For more information about this package, see the upstream documentation in
+/usr/share/doc/iperf-2.0.4.
+
+Cygwin port maintained by: Michael White mike9white22Z at yahooZ dot com remove Z and 9
+Please address all questions to the Cygwin mailing list at <cygwin@cygwin.com>
--- origsrc/iperf-2.0.4/CYGWIN-PATCHES/setup.hint	1969-12-31 19:00:00.000000000 -0500
+++ src/iperf-2.0.4/CYGWIN-PATCHES/setup.hint	2010-03-16 08:27:22.427685600 -0400
@@ -0,0 +1,5 @@
+category: Net
+requires: libgcc1 libstdc++6
+sdesc: "Internet Protocol bandwidth measuring tool"
+ldesc: "Iperf was developed by NLANR/DAST as a modern alternative for measuring maximum TCP and UDP bandwidth performance. Iperf allows the tuning of various parameters and UDP characteristics. Iperf reports bandwidth, delay jitter, datagram loss."
+
